ckanclient/__init__.py

```diff
@@ -175,9 +175,9 @@ def push_blob(self, resource):  # TODO async version with on_progress attr
             'fileExists': True,
         }
 
-        if not obj.get("actions"):  # File is already in storage
-            return result
+        if obj.get("actions"):  # file is not in storage
+            push_data_to_blob_storage(obj["actions"]["upload"], resource)
+            verify_upload(obj["actions"]["verify"], resource)
+            result["file_exists"] = False
```

Comment:
should it be `result['fileExists']` like here?
https://github.com/datopian/ckan-client-py/pull/22/files#diff-0587e0b25e7ddf38248a5d05e762027bbc2c3ee2e00f8896d2672560b68abd06R175

Reply:
Ow… sure. I forgot I opted for camel case to keep it in sync with the JS client! Gonna fix that. Tks : )